test/pica.js

```diff
@@ -39,4 +39,20 @@ describe('API', function () {
     });
   });
 
+  it('Resize with invalid size should fail with promise rejection', function () {
+    let src = new Canvas();
+
+    src.width = 1000;
+    src.height = 1000;
+
+    let to = new Canvas();
+
+    to.width = 0;
+    to.height = 0;
+
+    return _pica().resize(src, to).catch(err => {
```

Comment:
Missed something like: `.then(() => throw new Error('should throw error'))`

Because if code will NOT reject, existing assert will not be called at all, and test will report siccess.